spec/loophp/collection/CollectionSpec.php

```diff
@@ -671,6 +671,10 @@ public function it_can_contains(): void
         $this::fromIterable(range('A', 'C'))
             ->contains('C', 'unknown', 'A')
             ->shouldIterateAs([true]);
+
+        $this::fromIterable(['a' => 'b', 'c' => 'd'])
+            ->contains('d')
+            ->shouldIterateAs(['c' => true]);
```

Comment:
this is another unexpected behaviour I discovered from `contains`, it doesn't always return a list, but rather keeps the key of the searched element